packages/core/src/utilities/getTargetVolumeAndSpacingInNormalDir.ts

```diff
@@ -7,6 +7,12 @@ import getSpacingInNormalDirection from './getSpacingInNormalDirection';
 // One EPSILON part larger multiplier
 const EPSILON_PART = 1 + EPSILON;
 
+// Check if this is a primary volume
+// For now, that means it came from some sort of image loader, but
+// should be specifically designated.
+const isPrimaryVolume = (volume): boolean =>
+  volume.volumeId.indexOf(':') !== -1;
```

Comment:
can we add a method to `volumeLoader` to `get` all loaders and just extract the scope and use it here instead?